linchpin/hooks/built_ins/azure_boot_log/boot_log.yml

```diff
@@ -12,6 +12,7 @@
       vault_pass: "{{ vault_pass }}"
     register: auth_var_out
     delegate_to: localhost
+    no_log: "{{ not auth_debug }}"
```

Comment:
This isn't really specific to this PR, but why do we have an `auth_debug` and a `debug` variable?  They seem to do the same thing

Reply:
Because when debugging `auth_debug` module it exposes the credentials, but we need debug enabled in CI tests for other modules.

Reply:
The debug variable was originally created to hide credentials.  If there's tasks that don't expose credentials in which debug is set, we should remove them in another PR and consolidate to one variable.  We don't need to worry about that for this PR though